projects/wave-core-new/src/lib/project/notifications/notifications.component.html

```diff
@@ -0,0 +1,16 @@
+<mat-card *ngIf="notifications.length" [style.overflow]="'auto'" [style.height.px]="'300'">
+    <mat-list *ngFor="let notification of notifications" dense>
+        <mat-list-item>
+            <button class="deleteIcon" mat-icon-button (click)="removeCurrentNotification(notification)">
+                <mat-icon>clear</mat-icon>
```

Comment:
please use the trash-can icon we also use for deleting plots like histogram in the plot view

Reply:
d27ffd9b23a6bb4f382ea9cea14eb83f3fb9ba10